src/palette/qml/MuseScore/Palette/PaletteCellPropertiesDialog.qml

```diff
@@ -80,11 +80,11 @@ StyledDialogView {
                 id: repeater
 
                 model: [
-                    { title: qsTrc("palette", "X"), value: propertiesModel.xOffset, incrementStep: 1,
+                    { title: qsTrc("palette", "X"), value: propertiesModel.xOffset, minValue: -999, incrementStep: 1,
                         //: Abbreviation of "spatium"
                         measureUnit: qsTrc("global", "sp") },
-                    { title: qsTrc("palette", "Y"), value: propertiesModel.yOffset, incrementStep: 1, measureUnit: qsTrc("global", "sp") },
-                    { title: qsTrc("palette", "Content scale"), value: propertiesModel.scaleFactor, incrementStep: 0.1 }
+                    { title: qsTrc("palette", "Y"), value: propertiesModel.yOffset, minValue: -999, incrementStep: 1, measureUnit: qsTrc("global", "sp") },
```

Comment:
1. we should not use such large numbers for min/max
2. we should also specify maxValue for X/Y/scale

@bkunda what values should we use?